gitdash/pages/api/issues.ts

```diff
@@ -23,36 +23,23 @@ export default async function GetIssueDetails(
   // Get user data
   const userData = await octokit.request("GET /user");
 
-  // Get all repos
-  const repos = await octokit.request("GET /user/repos");
+  // Get all issues
+  const issues = await octokit.request("GET /issues");
```

Comment:
Does this work? It didn't work for us earlier and we spent like hours figuring this out.

Reply:
Yep you can hop into this branch and try it out! It must be because I've added the scopes for authentication in a previous PR.

Reply:
I can test in a while after I make a readme PR as well but if it's working because of the scope then I can 
merge it as well. Whatever works.